scripts/30-keyboard-layout-service.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+qvm-features-request supported-service.keyboard-layout=1
```

Comment:
`supported-service.` prefix is reserved for things controlled with `qvm-service` (aka `service.`-prefixed features).
Normally announcing supported features requires appropriate part in core-admin (like [here](https://github.com/QubesOS/qubes-core-admin/blob/master/qubes/ext/core_features.py#L28-L64)). Since this isn't the first time we need a simple feature support to announce that isn't strictly `qvm-service`, perhaps we need a generic mechanism for this? Maybe similar thing with `supported-feature.` prefix?

Reply:
Sure, that sounds perfectly fine.